tests/e2e/default/navigation/more-options-menu/offline-user.wdio-spec.js

```diff
@@ -0,0 +1,49 @@
+const commonPage = require('../../../../page-objects/default/common/common.wdio.page');
+const contactPage = require('../../../../page-objects/default/contacts/contacts.wdio.page');
+const reportPage = require('../../../../page-objects/default/reports/reports.wdio.page');
+const utils = require('../../../../utils');
+const seeder = require('./seeder');
+
+describe('Export enabled when there are items: messages, contacts, peope', async () => {
+  before(async () => {
+    await seeder.saveDocs();
+    await utils.createUsers([seeder.offlineUser]); //offline user
+    await seeder.newLogin(seeder.offlineUser);
+  });
+
+  it('- Message tab', async () => {
+    await commonPage.goToMessages();
+    await seeder.sendMessage();
+    expect(await (await commonPage.moreOptionsMenu()).isExisting()).to.be.false;    
+  });
+
+  it('- Contact tab: no contact selected', async () => {
```

Comment:
Whenever you have more than 1 test case about the same unit, you can group them inside a `describe(...)`, so you don't need to tag them with `- Contact tab .....`, because the reporter will print them with the `describe(..)` label. Also it's nice way to help us collapse a big chunk of code block by "topics" :) 

For example, consider this:

```
describe('More options menu component', () => {

   describe('Contact tab',  () => {
      it('should display edit option when user has permissions', () => { ..... });

      it('should not display delete option when user does not have permissions', () => { ..... });
   });
});
```

The reporter prints: 
```
>> More options menu component
>>    Contact tab
>>         should display edit option when user has permissions: PASSED
>>         should not display delete option when user does not have permissions: FAILED
```

Please apply across this file

Reply:
Real-life example: 
![Screen Shot 2022-12-22 at 10 18 05 am](https://user-images.githubusercontent.com/66472237/209048199-bb0221aa-a49f-4c48-8ef6-6b0d57954b2c.png)

In my opinion, it looks better